translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+      </trans-unit>
+      <trans-unit id="6088417909306773667" datatype="html">
+        <source>This space is not aligned with the current line.</source>
+        <target>Cette case n&apos;est pas alignée avec la ligne actuellement formée.</target>
+      </trans-unit>
+      <trans-unit id="6178824149031907459" datatype="html">
+        <source>Initial board and goal of the game</source>
+        <target>Plateau initial et but du jeu</target>
+      </trans-unit>
+      <trans-unit id="2613028380797438509" datatype="html">
+        <source>At Abalone, the goal of the game is to be the first player to push 6 opponent&apos;s pieces out of the board. Let us see how!</source>
+        <target>À l&apos;Abalone, le but du jeu est d&apos;être le premier joueur à pousser 6 pièces adverses en dehors du plateau. Voyons voir comment !</target>
+      </trans-unit>
+      <trans-unit id="4612562967450553112" datatype="html">
+        <source>Moving a piece</source>
+        <target>Déplacer une pièce</target>
+      </trans-unit>
+      <trans-unit id="980251877705717270" datatype="html">
+        <source>At each turn, move one, two, or three pieces either along their line, or on their side.
+        For your moves you can therefore choose between up to 6 directions.
+        The pieces you move must be aligned and consecutive, and the move should land on an empty space (except to push, we will see that later).
+        To make a move, click on one of your pieces, then click on an arrow to choose the move direction.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Dark, make any move!</source>
+        <target>Chaque tour, déplacez une, deux ou trois pièces, soit le long de leur alignement, soit par un pas de côté.
+        Pour vos déplacement vous avez donc au maximum à choisir parmi 6 directions.
+        Les trois pièces à déplacer doivent être alignées et immédiatement voisines et atterrir sur des cases vides (sauf pour pousser, ce que nous verrons plus tard).
+        Pour effectuer un déplacement, cliquez sur une de vos pièces, puis cliquez sur une flèche pour choisir sa direction.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez Foncé, faites n&apos;importe quel mouvement !</target>
+      </trans-unit>
+      <trans-unit id="3762527362373672599" datatype="html">
+        <source>Congratulations!</source>
+        <target>Bravo !</target>
+      </trans-unit>
+      <trans-unit id="272253201636921624" datatype="html">
+        <source>Pushing</source>
+        <target>Pousser</target>
+      </trans-unit>
+      <trans-unit id="3793007300702475909" datatype="html">
+        <source>To push one opponent piece, you must move at least two of your pieces.
+        To push two opponent pieces, you must move three of your pieces.
+        If one of your pieces is on the way, it will be impossible to push.
+        You cannot move more than three pieces.&lt;br/&gt;&lt;br/&gt;
```

Comment:
C'est moi qui ai dit "on the way" mais "in the way" est pas plus correct ?

Reply:
toutafait !